src/scripts/update_container_image.sh

```diff
@@ -13,19 +14,26 @@ else
     echo "Error: The update-container-image command requires either resource-file-path or resource-name to be specified."
     exit 1
 fi
+
 if [ -n "${CONTAINER_IMAGE_UPDATES}" ]; then
-    set -- "$@" ${CONTAINER_IMAGE_UPDATES}
+    IFS=" " read -a args -r <<< "${CONTAINER_IMAGE_UPDATES[@]}"
+    for arg in "${args[@]}"; do
+        set -- "$@" "$arg"
+    done
 fi
+
 if [ -n "${NAMESPACE}" ]; then
     set -- "$@" --namespace="${NAMESPACE}"
 fi
 if [ -n "${DRY_RUN}" ]; then
     set -- "$@" --dry-run "${DRY_RUN}"
 fi
-if [ "$SHOW_EKSCTL_COMMAND" == "1" ]; then
+if [ "$PARAM_SHOW_KUBECTL_COMMAND" == "1" ]; then
```

Comment:
Is this meant to be follow the orb tools 12 standard conventions?

This might need to be something like `KUBE_BOOL_SHOW_KUBECTL_COMMAND`

Reply:
The convention is `[ORB_NAME]_[VAR_TYPE]_VARIABLE_NAME` 

I just changed all of them 👍